### PR TITLE
master into 6.0 (error free)

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
Currently getting these types of errors 
image

**Comment by micahgodbolt** [comment]
okay, classnameFunction and styled and IStyleFunction have been moved to utilities! on it

**Comment by betrue-final-final** [comment]
I believe that this PR made the shadows darker. I rejected 6.0 in screener.io, fyi. Any reason why they were made darker?

**Comment by micahgodbolt** [comment]
Can you be specific as to which shadows? Probably something happened in the move from scss to getStyles.

**Comment by betrue-final-final** [comment]
They are here in screener. Callouts, menus and you name it. https://screener.io/v2/dashboard/ryQqxfIYZ.OfficeDev-office-ui-fabric-react/6.0

**Comment by micahgodbolt** [comment]
Yeah, I moved the callout box shadow mixin from sass to JS, and I never actually saw a color, which probably means I used the default color. Guess somewhere in the sass logic, the color is specified. I'll track it down and update. Thanks for the catch.

**Comment by betrue-final-final** [comment]
Excellent!

**Comment by micahgodbolt** [comment]
weird, turns out `@include ms-drop-shadow(0,0, 5px, 0px);` produces a drop shadow of `0 0 5px 0px rgba(0,0,0,0.4)`. Didn't see that opacity anywhere in the mixin. I'll try to track it down again.

**Comment by micahgodbolt** [comment]
haha...nerrrrrrr I'm dumb

```
@mixin ms-drop-shadow($x-offset: 0, $y-offset: 0, $blur: 5px, $spread: 0, $alpha: 0.4) {
  @include ms-box-shadow($x-offset, $y-offset $blur $spread rgba(0, 0, 0, $alpha));
}
```

**Comment by micahgodbolt** [comment]
hey, travis link is back!   hope that means our integration will be working properly again

File: packages/office-ui-fabric-react/src/components/Callout/Callout.styles.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function getStyles(props: ICalloutStyleProps): ICalloutStyles {
     root: [
       'ms-Callout',
       {
-        boxShadow: '0 0 5px 0px',
+        boxShadow: '0 0 5px 0px rgba(0,0,0,.4)',
```
Comment (betrue-final-final) [inline packages/office-ui-fabric-react/src/components/Callout/Callout.styles.tsx]: Noice!

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/Callout/Callout.styles.tsx]: yup..design detail down at the callout level will have a BIG impact on all the components that use it

**Comment by micahgodbolt** [comment]
drat...everything is reporting except for screener

**Comment by betrue-final-final** [comment]
Yeah, I'm getting a screener error.

**Comment by micahgodbolt** [comment]
yeah, saw that error. trying to repro locally. not sure what my change would break

**Comment by micahgodbolt** [comment]
yup, I can repro locally. diving in.

**Comment by micahgodbolt** [comment]
yup, somehow broke breadcrumbs test.... ON IT!

**Comment by micahgodbolt** [comment]
all visuals and CI are passing! Merging this in.

**Pull request merged**